tests/helpers.bash

```diff
@@ -21,11 +21,18 @@ function setup() {
     suffix=$(dd if=/dev/urandom bs=12 count=1 status=none | od -An -tx1 | sed -e 's, ,,g')
     TESTDIR=${BATS_TMPDIR}/tmp${suffix}
     rm -fr ${TESTDIR}
-    mkdir -p ${TESTDIR}/{root,runroot}
-
+    mkdir -p ${TESTDIR}/{root,runroot,sigstore,registries.d}
+    echo "default-docker:                                                           " >> ${TESTDIR}/registries.d/default.yaml
+    echo "  sigstore-staging: file://${TESTDIR}/sigstore                            " >> ${TESTDIR}/registries.d/default.yaml
+    echo "docker:                                                                   " >> ${TESTDIR}/registries.d/default.yaml
+    echo "  registry.access.redhat.com:                                             " >> ${TESTDIR}/registries.d/default.yaml
+    echo "    sigstore: https://access.redhat.com/webassets/docker/content/sigstore " >> ${TESTDIR}/registries.d/default.yaml
+    echo "  registry.redhat.io:                                                     " >> ${TESTDIR}/registries.d/default.yaml
+    echo "    sigstore: https://registry.redhat.io/containers/sigstore              " >> ${TESTDIR}/registries.d/default.yaml
```

Comment:
I think this would be more readable as:
```bash
    cat >${TESTDIR}/registries.d/default.yaml <<EOF
default-docker:
....
EOF
```
...but it's not worth resubmitting for.